src/components/form-attachment/link-dataset.vue

```diff
@@ -0,0 +1,104 @@
+<!--
+Copyright 2022 ODK Central Developers
+See the NOTICE file at the top-level directory of this distribution and at
+https://github.com/getodk/central-frontend/blob/master/NOTICE.
+
+This file is part of ODK Central. It is subject to the license terms in
+the LICENSE file found in the top-level directory of this distribution and at
+https://www.apache.org/licenses/LICENSE-2.0. No part of ODK Central,
+including this file, may be copied, modified, propagated, or distributed
+except according to the terms contained in the LICENSE file.
+-->
+<template>
+  <modal id="form-attachment-link-dataset" :state="state" :hideable="!awaitingResponse"
+    backdrop @hide="$emit('hide')">
+    <template #title>{{ $t('title') }}</template>
+    <template #body>
+      <div class="modal-introduction">
+          <p>
+            {{ $t('introduction[0]') }}
+            <template v-if="blobExists"> {{ $t('introduction[1]') }}</template>
```

Comment:
This is a great case for `<sentence-separator>`. That'd be something like:

```vue
<span>{{ $t('introduction[0]') }}</span>
<sentence-separator/>
<template v-if="blobExists">{{ $t('introduction[1]') }}</template>
```

That way, the only way a space character comes in is via `<sentence-separator>`.